change/@fluentui-react-table-8682f0cf-67c2-4f72-8a1a-fa9e4cbeb580.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26690&commitId=8a16ae73ff1ce50dd5aec46d2bd6e224b3a8799f&env=prod)

✅ There was 6 screenshots added, 0 screenshots removed, 1837 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table layout flex - truncate.default (disabled).chromium.png|0| Added|
|Table layout flex - truncate.false.chromium.png|0| Added|
|Table layout flex - truncate.true.chromium.png|0| Added|
|Table layout table - truncate.default (disabled).chromium.png|0| Added|
|Table layout table - truncate.false.chromium.png|0| Added|
|Table layout table - truncate.true.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>